dagster/pandas_kernel/__init__.py

```diff
@@ -97,3 +100,36 @@ def single_path_arg(input_name, path):
 
 def json_input(name):
     return create_json_input(name)
+
+
+# XXX(freiksenet): this should really go to main solid file
```

Comment:
Yeah definitely move this to dagster core before commit.

Reply:
So I should have been more clear.

It can't be moved there because dataframe_soild wraps the transform.

Reply:
I kinda feel input/output validation should be part of core solid functionality, then there would be no need for wrapping.